src/index.js

```diff
@@ -16,10 +16,15 @@ import RoverList from './containers/RoverList';
 import Accounts from './containers/Accounts/Base';
 import MissionControl from './containers/MissionControl';
 
-const reduxMiddleware = applyMiddleware(
-  thunk,
-  createLogger(),
-  promiseMiddleware(),
+/* eslint-disable no-underscore-dangle */
```

Comment:
If you don't want to have both eslint comment lines, you can use `eslint-disable-next-line` here